subprojects/docs/src/docs/release/notes.md

```diff
@@ -181,6 +181,37 @@ This allows configuration of PMD to run its analysis on more than one thread.
 
 See the [documentation](userguide/pmd_plugin.html#sec:pmd_conf_threads) for more information.
 
+### Configuration cache improvements
+
+#### Modification of system properties at the configuration time
+
+Changes to system properties made at the configuration time are now also applied when the configuration cache entry is reused.
+This includes adding a property, modifying or removing an existing one.
+The types used for property keys and values have to be [supported by the configuration cache](userguide/configuration_cache.html#config_cache:requirements:disallowed_types).
+
+#### Running external processes at the configuration time
+
+Running external processes at the configuration time with Java or Groovy APIs or with `exec` and `javaexec` Gradle APIs is
+now [disallowed](userguide/configuration_cache.html#config_cache:requirements:external_processes) if the configuration cache is enabled.
```

Comment:
We should say that before we ignored them and that was a correctness issue.

Reply:
Done